### PR TITLE
DBZ-4465 Change version under test to 1.9.0

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-4465

### CODE REVIEW

File: debezium-connect-rest-extension/src/test/java/io/debezium/kcrestextension/TestHelper.java [hunk 1]
```diff
@@ -48,8 +48,8 @@ public static void setupDebeziumContainer(String debeziumVersion) {
                 .withNetwork(NETWORK)
                 .withCopyFileToContainer(
                         MountableFile.forHostPath(
-                                "target/debezium-connect-rest-extension-1.8.0-SNAPSHOT.jar"),
-                        "/kafka/libs/debezium-kcd-rest-extension-1.8.0.jar")
+                                "target/debezium-connect-rest-extension-1.9.0-SNAPSHOT.jar"),
```
Comment (gunnarmorling) [inline debezium-connect-rest-extension/src/test/java/io/debezium/kcrestextension/TestHelper.java:51]: Can't we use `Module.version()` or something?

Comment (jpechane) [reply debezium-connect-rest-extension/src/test/java/io/debezium/kcrestextension/TestHelper.java:51]: @gunnarmorling Not sure about that, that's for https://issues.redhat.com/browse/DBZ-4466 to solve.

Comment (jpechane) [reply debezium-connect-rest-extension/src/test/java/io/debezium/kcrestextension/TestHelper.java:51]: And `Module.version()` is available for connector Maven modules.

Comment (gunnarmorling) [reply debezium-connect-rest-extension/src/test/java/io/debezium/kcrestextension/TestHelper.java:51]: Ok, thanks for logging the issue, we can handle it separately then.

**Pull request merged**